app/views/handlers/individual_response.py

```diff
@@ -842,3 +867,59 @@ def handle_post(self):
                 mobile_number=self._request_args.get("mobile_number"),
             )
         )
+
+
+class FulfilmentRequest:
+    def __init__(self, metadata: Mapping, mobile_number: Optional[str] = None):
+        self._metadata = metadata
+        self._mobile_number = mobile_number
+        self._fulfilment_type = "sms" if self._mobile_number else "postal"
+
+    def _get_individual_case_id_mapping(self) -> Mapping:
+        return (
+            {}
+            if self._metadata.get("case_type") == "SPG"
+            else {"individualCaseId": str(uuid4())}
+        )
+
+    def _get_mobile_number_mapping(self) -> Mapping:
+        return {"telNo": self._mobile_number} if self._mobile_number else {}
+
+    def _get_fulfilment_code(self) -> str:
+        fulfilment_codes = {
+            "sms": {
+                GB_ENG_REGION_CODE: "UACITA1",
+                GB_WLS_REGION_CODE: "UACITA2B",
+                GB_NIR_REGION_CODE: "UACITA4",
+            },
+            "postal": {
+                GB_ENG_REGION_CODE: "P_UAC_UACIP1",
+                GB_WLS_REGION_CODE: "P_UAC_UACIP2B",
+                GB_NIR_REGION_CODE: "P_UAC_UACIP4",
+            },
+        }
+
+        region_code = self._metadata["region_code"]
+        return fulfilment_codes[self._fulfilment_type][region_code]
+
+    @cached_property
+    def payload(self) -> bytes:
+        message = {
+            "event": {
+                "type": "FULFILMENT_REQUESTED",
+                "source": "QUESTIONNAIRE_RUNNER",
+                "channel": "EQ",
+                "dateTime": datetime.now(tz=tzutc()).isoformat(),
```

Comment:
Should this definitely be UTC? In the payload description there is nothing to denote it (i.e. it doesn't end in `Z` which would indicate UTC timezone

Reply:
As discussed, I confirmed this with Mark. Python by default won't show 'Z' for Zulu time, it would show the offset which is fine from RM's perspective.

Reply:
Sorry, should have been more specific, the string in the payload didn't contain any denotion of timezone or offset. If this is known by RM then that's fine, but ideally we should be explicit